### PR TITLE
Adds picket signs

### PR DESCRIPTION
**Pull request by unid15**
Sprites and idea from tg

Craftable by using a metal rod on 2 pieces of cardboard
Use a pen or a crayon to change the text

### CODE REVIEW

File: code/game/objects/items/picketsign.dm [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	force = 5
+	w_class = 4.0
+	attack_verb = list("bashed", "smacked")
+
+	var/label = ""
+	var/spam_flag
+
+/obj/item/weapon/picket_sign/attackby(obj/item/W, mob/user)
+	if(istype(W, /obj/item/weapon/pen) || istype(W, /obj/item/toy/crayon))
+		var/txt = stripped_input(user, "What would you like to write on the sign?", "Sign Label", null , 30)
```
Comment (ComicIronic) [inline code/game/objects/items/picketsign.dm]: This needs a check for adjacency after the input.

Comment (ComicIronic) [inline code/game/objects/items/picketsign.dm]: YOU NEED TO FIX THIS

Comment (PJB3005) [inline code/game/objects/items/picketsign.dm]: https://github.com/d3athrow/vgstation13/pull/5166#discussion_r34241664

Comment (PJB3005) [inline code/game/objects/items/picketsign.dm]: https://github.com/Unfit/vgstation13/pull/3

**Pull request merged**